lib/twilio-ruby/http/http_client.rb

```diff
@@ -6,7 +6,7 @@ module Twilio
   module HTTP
     class Client
       attr_accessor :adapter
-      attr_reader :timeout, :last_response, :last_request
+      attr_reader :timeout, :last_response, :last_request, :connection
```

Comment:
Why do we need connection here, when we are creating one in line 30, while making request ?

Reply:
The connection variable was always there, I made the variable readable to use it in unit testing